packages/fluentui/react-northstar/src/components/Tree/Tree.tsx

```diff
@@ -27,6 +27,7 @@ import {
   ComponentEventHandler,
 } from '../../types';
 import { hasSubtree, removeItemAtIndex, getSiblings, TreeContext, TreeRenderContextValue } from './utils';
+import * as keyboardKey from 'keyboard-key';
```

Comment:
is this used in the component? I don't see it, so you should be able to remove it. if it is use, pls try to find a way to move keyboard checks to the behavior

Reply:
Yeah, I moved to behavior and forgot to remove the import... But it's done now :)